components/template-cs-style-rules.tpl

```diff
@@ -136,6 +136,11 @@ body {
   color: var(--content-button-hover-color);
```

Comment:
I suggest that button text color is solid black by default. So it can be binded with the buttons color controller in Design Editor. But it needs discussion with the designer before applying it.